docs/modules/documentation/containers/dropdown/dropdown-docs.component.ts

```diff
@@ -1,835 +1,274 @@
-                            'vertical-bar-chart-2',
-                            'vertical-bullet-chart',
-                            'vertical-grip',
-                            'vertical-stacked-chart',
-                            'vertical-waterfall-chart',
-                            'video',
-                            'visits',
-                            'waiver',
-                            'wallet',
-                            'warning',
-                            'warning2',
-                            'washing-machine',
-                            'weather-proofing',
-                            'web-cam',
-                            'widgets',
-                            'windows-doors',
-                            'work-history',
-                            'workflow-tasks',
-                            'world',
-                            'wounds-doc',
-                            'wrench',
-                            'write-new',
-                            'write-new-document',
-                            'x-ray',
-                            'zoom-in',
-                            'zoom-out'
-                        ]
-                    },
-                    size: {
-                        type: 'string',
-                        enum: ['default', 'xs', 's', 'compact', 'l']
-                    },
-                    label: {
-                        type: 'string'
-                    },
-                    buttonText: {
-                        type: 'string'
-                    },
-                    noBorder: {
-                        type: 'boolean'
-                    },
-                    items: {
-                        type: 'object',
-                        properties: {
-                            optionOne: {
-                                type: 'string'
-                            },
-                            optionTwo: {
-                                type: 'string'
-                            },
-                            OptionThree: {
-                                type: 'string'
-                            },
-                            GroupOption: {
-                                type: 'object',
-                                properties: {
-                                    optionFour: {
-                                        type: 'string'
-                                    },
-                                    optionFive: {
-                                        type: 'string'
-                                    },
-                                    optionSix: {
-                                        type: 'string'
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            }
-        },
-        type: 'object'
-    };
+  
+    textDropdownHtml =`  <fd-popover [isDropdown]="true">
```

Comment:
This is nitpick-y of me and I apologize but there is some extra indentation in the example html on this page